backend/src/main/java/com/eyetrade/backend/mapper/PredictionMapper.java

```diff
@@ -0,0 +1,86 @@
+package com.eyetrade.backend.mapper;
+
+import com.eyetrade.backend.constants.GeneralConstants;
+import com.eyetrade.backend.constants.PredictionStatus;
+import com.eyetrade.backend.model.dto.PredictionDto;
+import com.eyetrade.backend.model.entity.Prediction;
+import com.eyetrade.backend.model.entity.PredictionCountOfUser;
+import com.eyetrade.backend.model.resource.prediction.PredictionCountResource;
+import com.eyetrade.backend.model.resource.prediction.PredictionResource;
+import com.eyetrade.backend.model.resource.prediction.PredictionsResource;
+import com.eyetrade.backend.utils.DateUtils;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+public class PredictionMapper {
```

Comment:
You can use mapstruct annotations, then there is no need to implement.
But it's okay!